### ISSUE TITLE
--get-format doesn't print output format when used with special names

### ISSUE THREAD

**Issue by endumiuz** [rg3/youtube-dl#4880]
It should output the actual format (303+141)
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v', '--get-format', '-f', 'bestvideo+bestaudio', 'http://youtu.be/Ihd6igXx0VY']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2015.02.04
[debug] Python version 2.7.8 - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg N-69233-g1f13348
[debug] Proxy map: {}
bestvideo+bestaudio
```

**Comment by dstftw** [comment]
Will be fixed in the next version. Thanks for the report.

**Comment by phihag** [comment]
I believe this issue to be fixed in youtube-dl 2015.02.09 and newer. See [our FAQ](https://yt-dl.org/update) if you need help updating.
